mifospay/src/main/res/layout/activity_si_details.xml

```diff
@@ -30,7 +30,7 @@
                     android:layout_height="wrap_content"
                     android:layout_margin="@dimen/marginItemsInSectionMedium"
                     android:textColor="@color/black"
-                    android:textSize="40sp"
+                    android:textSize="30sp"
```

Comment:
`30sp` seems the better choice. 40 was way too big xD

@haran2248 instead of hardcoded value please use/create a dimension resource

Reply:
Yeah im working on it